### ISSUE TITLE
Provide guidance on reporting issues

### ISSUE THREAD

**Issue by hidde** [w3c/wai-wcag-em#28]
Currently, WCAG-EM doesn't say a whole lot about how examples of 'not meeting WCAG' are to be included:

> Reports should include at least one example for each conformance requirement and WCAG 2.0 Success Criterion not met. It is also good practice for evaluators to indicate issues that occur repeatedly.

(from [step 5.a](https://www.w3.org/TR/WCAG-EM/#step5a))

I think WCAG conformance evaluations pose an opportunity to help teams improve the accessibility of their product. The helpfulness of an evaluation report to teams depends on how clearly problems (and solutions) are defined. If an evaluator makes it easy for teams to solve the problem, it is more likely the problem gets solved, and with that, more likely that accessibility for end users increases.

**Comment by hidde** [comment]
Possible directions: describe page and the part of it that is broken. Then describe what is broken about it (can't access menu with keyboard) and mention relevant SC that is not met because of that. Optionally, explain what causes it (eg href attr missing), who it impacts, how to fix it.

**Comment by mgifford** [comment]
I wanted to put in a related link from OpenACR https://github.com/GSA/openacr/issues/180
